.env.example

```
@@ -19,4 +19,4 @@ NEXT_PUBLIC_POSTHOG_HOST=
 DATABASE_URL=
 
 # RESEND API KEY FOR EMAILS
-NEXT_PUBLIC_RESEND_API_KEY=
\ No newline at end of file
+# NEXT_PUBLIC_RESEND_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)